climpyrical/.coverage

```diff
@@ -0,0 +1 @@
+!coverage.py: This is a private format, don't read it directly!{"lines":{"/home/nannau/temp/climpyrical/climpyrical_master/climpyrical/climpyrical/__init__.py":[1],"/home/nannau/temp/climpyrical/climpyrical_master/climpyrical/climpyrical/tests/__init__.py":[1],"/home/nannau/temp/climpyrical/climpyrical_master/climpyrical/climpyrical/tests/test_gridding.py":[1,14,15,18,19,20,21,24,25,27,28,30,31,32,35,36,37,39,40,51,52,54,55,56,57,58,59,60,71,77,79,80,84,85,86,87,88,92,93,94,95,96,97,98,102,103,105,106,107,125,126,129,130,131,132,133,136,137,138,139,140,150,151,152,155,156,158,159,160,161,162,163,174,175,176,182,183,185,187,188,189,190,191,194,195,196,197,198,201,202,203,204,205,219,220,223,224,225,226,227,239,240,241,243,244,246,247,249,250,252,253,256,257,259,260,261,262,263,264,265,276,277,278,279,280,281,282,285,286,288,289,44,47,48,45,64,67,68,65,73,74,118,121,122,145,167,170,171,168,179,212,215,216,213,232,233,234,235,269,270,272,273,293,295,296,297,298,301],"/home/nannau/temp/climpyrical/climpyrical_master/climpyrical/climpyrical/gridding.py":[1,2,3,6,36,88,128,183,185,186,187,188,189,190,191,225,261,297,333,364,406,20,21,22,24,27,25,28,31,29,33,57,58,60,62,64,66,68,71,78,85,73,75,80,82,120,121,122,123,125,148,149,150,217,152,153,156,160,161,162,169,170,171,177,218,219,220,222,243,244,245,247,249,252,255,258,253,250,248,256,279,280,281,282,283,284,286,287,292,293,285,289,290,294,314,316,317,318,320,321,323,325,326,327,330,358,359,360,361,382,386,387,390,395,400,403,383,384,389,396,397,401,391,392,434,437,442,447,476,478,449,452,455,459,460,464,465,468,472],"/home/nannau/temp/climpyrical/climpyrical_master/climpyrical/climpyrical/mask.py":[],"/home/nannau/temp/climpyrical/climpyrical_master/climpyrical/climpyrical/tests/test_mask.py":[]}}
```

Comment:
I don't know what this file is about, but I have serious doubts that paths to your home directory belong in the repository.

Reply:
Thanks. That was accidentally added.